pkg/cmd/kafka/create/create.go

```diff
@@ -509,15 +511,22 @@ func promptKafkaPayload(opts *options, constants *remote.DynamicServiceConstants
 		Help:    f.Localizer.MustLocalize("kafka.create.input.name.help"),
 	}
 
-	// Get the list of enterprise clusters in the users organization if there are any
-	err := setEnterpriseClusterList(opts)
+	answers := &promptAnswers{}
+
+	err := survey.AskOne(promptName, &answers.Name, survey.WithValidator(validator.ValidateName), survey.WithValidator(validator.ValidateNameIsAvailable))
+	if err != nil {
+		return nil, err
+	}
+
+	// Get the list of enterprise clusters in the users organization if there are any, creates a map of cluster ids to names to include names in the prompt
+	err = setEnterpriseClusterList(opts)
```

Comment:
no need to prompt for cluster if opts.cluster_id has been set

Reply:
This is happening when we are in interactive mode meaning the `cluster_id` has not been set anyway